index.html

```diff
@@ -1,116 +1,229 @@
+    <main>
+      <section
+        class="bg-text-clip h-screen bg-[url('assets/desk.png')] bg-contain bg-opacity-10"
+      >
+        <div
+          class="px-[10%] bg-[#5f5ff50] h-screen flex justify-center flex-col gap-5"
+        >
+          <div class="text-[#ec5242]">
+            <h2 class="text-[25px] md:text-[50px]">"Hello! Sharing world"</h2>
+            <h1 class="text-[40px] md:text-[80px] (lato black)">
+              The Grape Escape<br />Wine Tasting and Braai
+            </h1>
+          </div>
+          <div
+            class="bg-[#d3d3d3] text-[#ec5242] w-[100%] md:w-[65%] px-[5%] py-[1%] border-solid border-2 border-[fff]"
+          >
+            <p>
+              Come raise your glass with us on yet another one of our annual
+              parties where we share inside information with future Scientists
+              on how to brew the finest wine in the world,
+            </p>
+          </div>
+          <div class="text-[#272a31]">
+            <h2 class="text-[25px] md:text-[40px] text-[#272a31]">
+              2023.March.17(Fri)~19(Sun)
+            </h2>
+            <p>@Stellenbosch Wine farms, Cape Town</p>
+          </div>
+        </div>
+      </section>
 
-  <!--Images -->
-  <div class="images">
-    <div class="image">
-      <img src="./assets/speaker_03.png" alt="Image 1">
-      <div class="image-text">
-        <h4>SohYeong Noh</h4>
-        <p>Chemistry Proffessor, University of Cape Town</p>
-      </div>
-    </div>
-    <div class="image">
-      <img src="./assets/speaker_01.png" alt="Image 2">
-      <div class="image-text">
-        <h4>Yochai Benkler</h4>
-        <p>Grape Farm Owner, South Africa</p>
-      </div>
-    </div>
-    <div class="image">
-      <img src="./assets/speaker_04.png" alt="Image 3">
-      <div class="image-text">
-        <h4>Moshe Khosi</h4>
-        <p>Description of Image 3</p>
-      </div>
-    </div>
-    <div class="image">
-      <img src="./assets/speaker_05.png" alt="Image 3">
-      <div class="image-text">
-        <h4>Teboho Nchai</h4>
-        <p>Liquor lake all round winner 2022</p>
+      <!-- main program -->
+
+      <section
+        class="bg-[#272a31] py-[5%] px-[3%] md:px-[10%] bg-[url('assets/program-bg.png')] text-center text-white flex flex-col items-center"
+      >
+        <h2 class="text-center text-white text-[20px]">Main program</h2>
+        <hr class="w-[40px] bg-[#ec5242] h-[3px]" />
+        <div
+          class="ml-[3%] mt-[50px] pb-[100px] grid gap-2 md:gap-5 grid-cols-1 md:grid-cols-5 justify-items-center"
+        >
+          <div
+            class="flex bg-[#5f5f5f50] p-[20px] md:flex-col flex-row gap-4 justify-center items-center"
+          >
+            <img src="assets/program_icon_01.png" alt="desk-logo" />
+            <h2 class="text-[20px] text-[#ec5242]">Lecture</h2>
+            <p class="text-[12px]" id="project-text">
+              Listen to the speakers from various countries about the messages
```

Comment:
- [ ] Kindly follow the design guideline you did not implement more the feature speakers.
- [ ] Mobile Design for home page

Your Design
<img width="506" alt="Screenshot 2023-03-18 at 15 00 41" src="https://user-images.githubusercontent.com/81444338/226142749-0ac28034-b492-4801-b37b-8e48aabbc265.png">

Behance Design
<img width="1440" alt="Screenshot 2023-03-18 at 15 02 21" src="https://user-images.githubusercontent.com/81444338/226142792-765468da-9916-4e9c-b4f6-e43747b2be1d.png">